src/components/molecules/NavBar/Navbar.styles.ts

```diff
@@ -1,19 +0,0 @@
-import styled from "styled-components";
-
-type RadioWrapperProps = {
-  showRadioPopover: boolean;
-};
-export const RadioWrapper = styled.div<RadioWrapperProps>`
-  position: absolute;
-  top: 100%;
-  right: 0;
-  visibility: ${({ showRadioPopover }) =>
-    showRadioPopover ? "visible" : "hidden"};
-  user-select: ${({ showRadioPopover }) =>
-    showRadioPopover ? "auto" : "none"};
-`;
-
-export const RadioTrigger = styled.div`
-  position: relative;
-  padding: 0.5em;
-`;
```

Comment:
[note] Moved to `src/components/molecules/NavbarRadio/NavbarRadio.scss`

Reply:
Looks like the `visibility`/`user-select` styles were lost from this and turned into inline styles. We want to keep them in the scss file, we can use the BEM modifier pattern for this.

Reply:
These have been moved to NavbarRadio and they are inline styles.

Reply:
Yeah, that’s what I said above.. but why are they inline styles, and why aren’t they scss anymore in that file?

Reply:
Because the styles are conditional and I removed the styled component to keep the code consistent, I also remember talking about removing styled components because it was added against our will.

Reply:
Have you seen our BEM variant patterns, and using the classNames helper to handle the conditional aspect of adding them/not in the component?